### PR TITLE
Map StuctureFeature partly

### PR DESCRIPTION
**Pull request by falseresync**

### CODE REVIEW

File: mappings/net/minecraft/world/gen/feature/StructureFeature.mapping [hunk 1]
```diff
@@ -3,19 +3,27 @@ CLASS net/minecraft/class_3195 net/minecraft/world/gen/feature/StructureFeature
...
 	FIELD field_24862 STRUCTURE_TO_GENERATION_STEP Ljava/util/Map;
+	FIELD field_24863 codec Lcom/mojang/serialization/Codec;
 	METHOD method_13149 getMonsterSpawns ()Ljava/util/List;
 	METHOD method_14015 locateStructure (Lnet/minecraft/class_4538;Lnet/minecraft/class_5138;Lnet/minecraft/class_2338;IZJLnet/minecraft/class_5314;)Lnet/minecraft/class_2338;
 		ARG 5 skipExistingChunks
 	METHOD method_14016 getStructureStartFactory ()Lnet/minecraft/class_3195$class_3774;
 	METHOD method_14019 getName ()Ljava/lang/String;
 	METHOD method_14026 shouldStartAt (Lnet/minecraft/class_2794;Lnet/minecraft/class_1966;JLnet/minecraft/class_2919;IILnet/minecraft/class_1959;Lnet/minecraft/class_1923;Lnet/minecraft/class_3037;)Z
 	METHOD method_16140 getCreatureSpawns ()Ljava/util/List;
+	METHOD method_27218 getStartingPosition (Lnet/minecraft/class_5314;JLnet/minecraft/class_2919;II)Lnet/minecraft/class_1923;
```
Comment (haykam821) [inline mappings/net/minecraft/world/gen/feature/StructureFeature.mapping]:
```suggestion
	METHOD method_27218 getStartPos (Lnet/minecraft/class_5314;JLnet/minecraft/class_2919;II)Lnet/minecraft/class_1923;
```

**Review by haykam821 — CHANGES_REQUESTED** [review]

**Review by liach — APPROVED** [review]

**Review by Runemoro — APPROVED** [review]

File: mappings/net/minecraft/world/gen/feature/StructureFeature.mapping [hunk 2]
```diff
@@ -3,19 +3,27 @@ CLASS net/minecraft/class_3195 net/minecraft/world/gen/feature/StructureFeature
...
+	FIELD field_24863 codec Lcom/mojang/serialization/Codec;
 	METHOD method_13149 getMonsterSpawns ()Ljava/util/List;
 	METHOD method_14015 locateStructure (Lnet/minecraft/class_4538;Lnet/minecraft/class_5138;Lnet/minecraft/class_2338;IZJLnet/minecraft/class_5314;)Lnet/minecraft/class_2338;
 		ARG 5 skipExistingChunks
 	METHOD method_14016 getStructureStartFactory ()Lnet/minecraft/class_3195$class_3774;
 	METHOD method_14019 getName ()Ljava/lang/String;
 	METHOD method_14026 shouldStartAt (Lnet/minecraft/class_2794;Lnet/minecraft/class_1966;JLnet/minecraft/class_2919;IILnet/minecraft/class_1959;Lnet/minecraft/class_1923;Lnet/minecraft/class_3037;)Z
 	METHOD method_16140 getCreatureSpawns ()Ljava/util/List;
+	METHOD method_27218 getStartPos (Lnet/minecraft/class_5314;JLnet/minecraft/class_2919;II)Lnet/minecraft/class_1923;
+	METHOD method_28656 createStructureStart (IILnet/minecraft/class_3341;IJ)Lnet/minecraft/class_3449;
```
Comment (Gegy) [inline mappings/net/minecraft/world/gen/feature/StructureFeature.mapping:15]: For consistency with `getStartPos`, would it make sense to elide the `Structure` prefix here? (or alternatively add the prefix to `getStartPos`?)

Comment (falseresync) [reply mappings/net/minecraft/world/gen/feature/StructureFeature.mapping:15]: Hm, possibly, need a third person to conclude

Comment (Runemoro) [reply mappings/net/minecraft/world/gen/feature/StructureFeature.mapping:15]: Yeah, it should just be `createStart`.

Comment (liach) [reply mappings/net/minecraft/world/gen/feature/StructureFeature.mapping:15]:
```suggestion
	METHOD method_28656 createStart (IILnet/minecraft/class_3341;IJ)Lnet/minecraft/class_3449;
```

**Comment by shartte** [comment]
I think this is superseded by #1552

**Pull request closed without merging**